sources/core/Xenko.Core/Diagnostics/Logger.cs

```diff
@@ -10,6 +10,7 @@ namespace Xenko.Core.Diagnostics
     /// </summary>
     public abstract partial class Logger : ILogger
     {
+        private static object _lock = new object();
```

Comment:
Is static necessary here ?

Reply:
There are other instances of logger, and the purpose of the lock is to make sure calls to log are single threaded across all instances.